documentation/release_notes.markdown

```diff
@@ -4,87 +4,271 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.0
```

Comment:
As the PR stands right now, there would be a roll up of all of the Puppet Server 1.x and the Puppet Server 2.1 release but the Puppet Server 2.0 notes would be missing, which would seem a little strange.

I'd be good with the separate file names in the repo to help avoid merge conflicts if we want only a subset of the notes to appear on each branch - or just doing a full roll-up of the notes that we try to keep in sync on both branches (although that could get pretty long over time).  

I think it would be better to separate out the question of how we want to maintain the release notes into a separate ticket so we don't block this PR indefinitely.  My preference would be to just leave the release notes file as is on master as part of this mergeup but if yall are good with it as it stands right now, I would be as well.  Mostly just want to get this merged soon to unblock other work which is dependent upon it.  @nfagerlund - any strong preferences?